tests/e2e/specs/proposal.spec.js

```diff
@@ -1,8 +1,9 @@
 /* eslint-disable ui-testing/no-disabled-tests */
-import { phrasesList } from '../utils';
+import { phrasesList, getTimeUntilVoteClose } from '../utils';
 
 describe('Make Proposal Tests', () => {
   let startTime;
+  const timeoutWindow = 120_000;
```

Comment:
if you're going to bind in this scope, might as well be named `timeout`

please sync with @rabi-siddique on how to define timeout consistently across this repos.